extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php

```diff
@@ -64,4 +65,68 @@ protected function hasNonPortableInfoBoxes() {
 		// TODO: Actually check for non portable infoboxes
 		return true;
 	}
+
+	protected function buildSurveyUrl() {
+		global $wgContLang;
+
+		$context = $this->getContext();
+		$userName = $context->getUser()->getName();
+		$userLanguage = $context->getLanguage();
+
+		$lang = $userLanguage->mCode;
+		$supportedLanguages = ['en', 'de', 'es', 'fr', 'it', 'ja', 'pl', 'pt-br', 'ru', 'zh'];
```

Comment:
It's a bummer that this list have to exist twice: here, and inside the googleFormUrl.
How about you declare just an associative array:
$masks = array(
)